### ISSUE TITLE
[Bug]: Error running install.py requirement: mediapipe==0.9.1.0

### ISSUE THREAD

**Issue by AixioX** [Mikubill/sd-webui-controlnet#871]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

Hi there, 

Can I please have some help of fixing my Controlnet issue as below... ControlNet just does't work when I use webUI after I update to the laster version. Device: MBP M2max 64ram, Venture 13.2.1... I'm totally a rookie about coding, but I guess I'll need to install mediapipe==0.9.1.0 to fix the issue? I've searched up online, but not many useful information. Many thanks! 

-------------------------------
Installing requirements for Web UI
Error running install.py for extension extensions/sd-webui-controlnet.
Command: "/Users/alvinxiong/stable-diffusion-webui/venv/bin/python3.10" "extensions/sd-webui-controlnet/install.py"
Error code: 1
stdout: Installing sd-webui-controlnet requirement: mediapipe==0.9.1.0

stderr: Traceback (most recent call last):
  File "/Users/alvinxiong/stable-diffusion-webui/extensions/sd-webui-controlnet/install.py", line 10, in <module>
    launch.run_pip(f"install {lib}", f"sd-webui-controlnet requirement: {lib}")
  File "/Users/alvinxiong/stable-diffusion-webui/launch.py", line 137, in run_pip
    return run(f'"{python}" -m pip {args} --prefer-binary{index_url_line}', desc=f"Installing {desc}", errdesc=f"Couldn't install {desc}")
  File "/Users/alvinxiong/stable-diffusion-webui/launch.py", line 105, in run
    raise RuntimeError(message)
RuntimeError: Couldn't install sd-webui-controlnet requirement: mediapipe==0.9.1.0.
Command: "/Users/alvinxiong/stable-diffusion-webui/venv/bin/python3.10" -m pip install mediapipe==0.9.1.0 --prefer-binary
Error code: 1
stdout: <empty>
stderr: ERROR: Could not find a version that satisfies the requirement mediapipe==0.9.1.0 (from versions: none)
ERROR: No matching distribution found for mediapipe==0.9.1.0

[notice] A new release of pip is available: 23.0 -> 23.1
[notice] To update, run: pip install --upgrade pip

Launching Web UI with arguments: --upcast-sampling --no-half-vae --use-cpu interrogate
Warning: caught exception 'Torch not compiled with CUDA enabled', memory monitor disabled
No module 'xformers'. Proceeding without it.
==============================================================================
You are running torch 1.12.1.
The program is tested to work with torch 1.13.1.
To reinstall the desired version, run with commandline flag --reinstall-torch.
Beware that this will cause a lot of large files to be downloaded, as well as
there are reports of issues with training tab on the latest version.

Use --skip-version-check commandline argument to disable this check.

### Steps to reproduce the problem

1. Go to Terminal 
2. Type: cd ~/stable-diffusion-webui;./webui.sh 
3. Terminal window show: Python Launch.py 80x24
4. webUI is normal, ControlNet shows normal, but when generate image, controlNet doesn't work. For example, Openpose, can see the stickman in CN window, check the setting, correct, but the generated image done't have the same/similar pose as the stickman. Have tested few hours.... 

### What should have happened?

ControlNet openpose, the generated image should have same or similar pose as the stickman 

### Commit where the problem happens

Installing requirements for Web UI
Error running install.py for extension extensions/sd-webui-controlnet.
Command: "/Users/alvinxiong/stable-diffusion-webui/venv/bin/python3.10" "extensions/sd-webui-controlnet/install.py"
Error code: 1
stdout: Installing sd-webui-controlnet requirement: mediapipe==0.9.1.0

stderr: Traceback (most recent call last):
  File "/Users/alvinxiong/stable-diffusion-webui/extensions/sd-webui-controlnet/install.py", line 10, in <module>
    launch.run_pip(f"install {lib}", f"sd-webui-controlnet requirement: {lib}")
  File "/Users/alvinxiong/stable-diffusion-webui/launch.py", line 137, in run_pip
    return run(f'"{python}" -m pip {args} --prefer-binary{index_url_line}', desc=f"Installing {desc}", errdesc=f"Couldn't install {desc}")
  File "/Users/alvinxiong/stable-diffusion-webui/launch.py", line 105, in run
    raise RuntimeError(message)
RuntimeError: Couldn't install sd-webui-controlnet requirement: mediapipe==0.9.1.0.
Command: "/Users/alvinxiong/stable-diffusion-webui/venv/bin/python3.10" -m pip install mediapipe==0.9.1.0 --prefer-binary
Error code: 1
stdout: <empty>
stderr: ERROR: Could not find a version that satisfies the requirement mediapipe==0.9.1.0 (from versions: none)
ERROR: No matching distribution found for mediapipe==0.9.1.0

[notice] A new release of pip is available: 23.0 -> 23.1
[notice] To update, run: pip install --upgrade pip

Launching Web UI with arguments: --upcast-sampling --no-half-vae --use-cpu interrogate
Warning: caught exception 'Torch not compiled with CUDA enabled', memory monitor disabled
No module 'xformers'. Proceeding without it.
==============================================================================
You are running torch 1.12.1.
The program is tested to work with torch 1.13.1.
To reinstall the desired version, run with commandline flag --reinstall-torch.
Beware that this will cause a lot of large files to be downloaded, as well as
there are reports of issues with training tab on the latest version.

Use --skip-version-check commandline argument to disable this check.

### What browsers do you use to access the UI ?

Apple Safari

### Command Line Arguments

```Shell
cd ~/stable-diffusion-webui;./webui.sh
```

### Console logs

```Shell
Last login: Tue Apr 18 12:46:59 on ttys000

The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.
AX-MBP14-M2:~ alvinxiong$ cd ~/stable-diffusion-webui;./webui.sh 

################################################################
Install script for stable-diffusion + Web UI
Tested on Debian 11 (Bullseye)
################################################################

################################################################
Running on alvinxiong user
################################################################

################################################################
Repo already cloned, using it as install directory
################################################################

################################################################
Create and activate python venv
################################################################

################################################################
Launching launch.py...
################################################################
Python 3.10.10 (main, Feb 16 2023, 02:49:39) [Clang 14.0.0 (clang-1400.0.29.202)]
Commit hash: 0cc0ee1bcb4c24a8c9715f66cede06601bfc00c8
Installing requirements for Web UI
Error running install.py for extension extensions/sd-webui-controlnet.
Command: "/Users/alvinxiong/stable-diffusion-webui/venv/bin/python3.10" "extensions/sd-webui-controlnet/install.py"
Error code: 1
stdout: Installing sd-webui-controlnet requirement: mediapipe==0.9.1.0

stderr: Traceback (most recent call last):
  File "/Users/alvinxiong/stable-diffusion-webui/extensions/sd-webui-controlnet/install.py", line 10, in <module>
    launch.run_pip(f"install {lib}", f"sd-webui-controlnet requirement: {lib}")
  File "/Users/alvinxiong/stable-diffusion-webui/launch.py", line 137, in run_pip
    return run(f'"{python}" -m pip {args} --prefer-binary{index_url_line}', desc=f"Installing {desc}", errdesc=f"Couldn't install {desc}")
  File "/Users/alvinxiong/stable-diffusion-webui/launch.py", line 105, in run
    raise RuntimeError(message)
RuntimeError: Couldn't install sd-webui-controlnet requirement: mediapipe==0.9.1.0.
Command: "/Users/alvinxiong/stable-diffusion-webui/venv/bin/python3.10" -m pip install mediapipe==0.9.1.0 --prefer-binary
Error code: 1
stdout: <empty>
stderr: ERROR: Could not find a version that satisfies the requirement mediapipe==0.9.1.0 (from versions: none)
ERROR: No matching distribution found for mediapipe==0.9.1.0

[notice] A new release of pip is available: 23.0 -> 23.1
[notice] To update, run: pip install --upgrade pip




Launching Web UI with arguments: --upcast-sampling --no-half-vae --use-cpu interrogate
Warning: caught exception 'Torch not compiled with CUDA enabled', memory monitor disabled
No module 'xformers'. Proceeding without it.
==============================================================================
You are running torch 1.12.1.
The program is tested to work with torch 1.13.1.
To reinstall the desired version, run with commandline flag --reinstall-torch.
Beware that this will cause a lot of large files to be downloaded, as well as
there are reports of issues with training tab on the latest version.

Use --skip-version-check commandline argument to disable this check.
==============================================================================
Loading weights [9aba26abdf] from /Users/alvinxiong/stable-diffusion-webui/models/Stable-diffusion/Deliberate_v2.safetensors
Creating model from config: /Users/alvinxiong/stable-diffusion-webui/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying cross attention optimization (InvokeAI).
Textual inversion embeddings loaded(0): 
Textual inversion embeddings skipped(6): dblx, 21charturnerv2, CGI_Animation, kc-v4-5000, remix, art by midjourney
Model loaded in 1.7s (create model: 0.5s, apply weights to model: 0.5s, apply half(): 0.3s, move model to device: 0.4s).
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
```

**Comment by lllyasviel** [comment]
fixed https://github.com/Mikubill/sd-webui-controlnet/pull/873
update and completely restart including terminal

**Comment by rabbit318** [comment]
> fixed #873
> update and completely restart including terminal

I have the same issue. What should I update?
